rlclientlib/logger/event_logger.cc

```diff
@@ -3,6 +3,17 @@
 #include "err_constants.h"
 #include "time_helper.h"
 namespace reinforcement_learning { namespace logger {
+  content_encoding_enum to_content_encoding_enum(const char *content_encoding)
+  {
+    if (std::strcmp(content_encoding, value::CONTENT_ENCODING_IDENTITY) == 0)
+    {
+      return content_encoding_enum::IDENTITY;
+    }
+    else
+    {
+      return content_encoding_enum::ZSTD_AND_DEDUP;
```

Comment:
If `content_encoding` is not a known value you will get `content_encoding_enum::ZSTD_AND_DEDUP`. Does it make sense to fail instead? If someone misconfigures this, I think it's not intuitive how it is

Reply:
Good catch, we should either fail or pick identity.